### PR TITLE
Edits to five short indexing pages as part of site review

### PR DESCRIPTION
**Pull request by matthewhelmke**
This PR contains simple edits to the following indexing pages:
- Bloom filter
- Inverted index
- Native text index
- Range index
- Timestamp index

This is an ongoing project from the documentation team and there will be many related PRs with edits to different sections to help with consistency, clarity, and manner of expression in the docs.

### CODE REVIEW

**Comment by matthewhelmke** [comment]
This seems to have pulled in some earlier commits. I'm not sure why.

File: basics/indexing/bloom-filter.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
+---
+description: This page describes configuring the bloom filter for Apache Pinot
+---
+
 # Bloom Filter
```
Comment (kelseiv) [inline basics/indexing/bloom-filter.md]:
```suggestion
# Bloom filter
```

File: basics/indexing/bloom-filter.md [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
 
-```
+```sql
 SELECT COUNT(*) 
 FROM baseballStats 
 WHERE playerID = 12345
 ```
 
-There are 3 parameters to configure the Bloom Filter:
+There are 3 parameters to configure the bloom filter:
```
Comment (kelseiv) [inline basics/indexing/bloom-filter.md:17]: Good example of where we can start making the structure more clear and modular.
For each index doc, we can add a procedural header for "Configure the xyz filter" and rewrite explicit steps to complete.
In future, we could regroup similar content into another chunk, perhaps "Configure the table index" where it makes sense to do so

File: basics/indexing/bloom-filter.md [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
 
-* `fpp`: False positive probability of the bloom filter (from `0` to `1`, `0.05` by default). The lower the `fpp` , the higher accuracy the bloom filter has, but it will also increase the size of the bloom filter.
-* `maxSizeInBytes`: Maximum size of the bloom filter (unlimited by default). If a certain `fpp` generates a bloom filter larger than this size, we will increase the `fpp` to keep the bloom filter size within this limit.
-* `loadOnHeap`: Whether to load the bloom filter using heap memory or off-heap memory (`false` by default).
+* **`fpp`**: False positive probability of the bloom filter (from `0` to `1`, `0.05` by default). The lower the `fpp` , the higher accuracy the bloom filter has, but it will also increase the size of the bloom filter.
```
Comment (kelseiv) [inline basics/indexing/bloom-filter.md]:
```suggestion
* **`fpp`**: False positive probability of the bloom filter (from `0` to `1`, `0.05` by default). A lower `fpp` increase both accuracy and size of the bloom filter.
```

File: basics/indexing/bloom-filter.md (hunk 2 above)
Comment (kelseiv) [reply basics/indexing/bloom-filter.md:17]: 1. Set the false positive probability `fpp` for the bloom filter between `0` to `1` (`0.05` by default). As the `fpp` decreases, the bloom filter size and accuracy increases.
...

File: basics/indexing/bloom-filter.md [hunk 4]
```diff
@@ -1,22 +1,26 @@
...
+* **`maxSizeInBytes`**: Maximum size of the bloom filter (unlimited by default). If a `fpp` setting generates a bloom filter larger than this size, using this setting will increase the `fpp` to keep the bloom filter size within this limit.
```
Comment (kelseiv) [inline basics/indexing/bloom-filter.md:20]: 2. Set the maximum size of the bloom filter `maxSizeInBytes` (unlimited by default)....

File: basics/indexing/bloom-filter.md [hunk 5]
```diff
@@ -1,22 +1,26 @@
...
+* **`loadOnHeap`**: Whether to load the bloom filter using heap memory or off-heap memory (`false` by default).
```
Comment (kelseiv) [inline basics/indexing/bloom-filter.md:21]: 3. Specify whether to load the bloom filter....

File: basics/indexing/native-text-index.md [hunk 6]
```diff
@@ -1,38 +1,32 @@
...
+# Native text index
 
-## History Of Text Indexing And Search in Pinot
+Pinot supports text indexing and search by building Lucene indices as _sidecars_ to the main Pinot segments. While this is a great technique, it essentially limits the avenues of optimizations that can be done for Pinot specific use cases of text search.
 
-Pinot supports text indexing and search by building Lucene indices as "sidecars" to the main Pinot segments. While this is a great technique, it essentially limits the avenues of optimizations that can be done for Pinot specific use cases of text search.
+## How is Pinot different?
 
-## How Is Pinot Different?
+Pinot, like any other database/OLAP engine, does not need to conform to the entire full text search domain-specific language (DSL) that is traditionally used by full-text search (FTS) engines like ElasticSearch and Solr. In traditional SQL text search use cases, the majority of text searches belong to one of three patterns: prefix wildcard queries (like `pino*`), postfix or suffix wildcard queries (like `*inot`), and term queries (like `pinot`).
```
Comment (kelseiv) [inline basics/indexing/native-text-index.md]:
```suggestion
Pinot, like any other database/OLAP engine, does not need to conform to the entire full text search domain-specific language (DSL) that is traditionally used by full-text search (FTS) engines like ElasticSearch and Solr. In traditional SQL text search use cases, the majority of text searches belong to one of three patterns: prefix wildcard queries (like `pino*`), suffix wildcard queries (like `*inot`), and term queries (like `pinot`).
```

File: basics/indexing/native-text-index.md [hunk 7]
```diff
@@ -1,38 +1,32 @@
...
-## Real Time Indexing And Search
-
-A new feature that native text indices support are real time text search. For REALTIME tables, native text indices allow data to be indexed in memory in the text index, while concurrently supporting text searches on the same index.
-
-Historically, most text indices depend on the in memory text index being written to first and then sealed, before searches are possible. This limits the freshness of the search, being near real time at best.
-
-Native text indices come with a custom in memory text index, which allows for real time indexing and search.
+Native text indices come with a custom in-memory text index, which allows for real-time indexing and search.
 
 ## Searching Native Text Indices
```
Comment (kelseiv) [inline basics/indexing/native-text-index.md:27]: are we using the term indices intentionally here, or do we want to say indexes?

**Comment by kelseiv** [comment]
> This seems to have pulled in some earlier commits. I'm not sure why.

Interesting. Did you `git pull` right before creating the PR? if not, try that next time, if so, I imagine it's something in gitbook we should test with @noramullen1 

I noticed 3 of the open PRs had changes across the PRs

File: basics/indexing/native-text-index.md (hunk 7 above)
Comment (matthewhelmke) [reply basics/indexing/native-text-index.md:27]: I thought it was intentional, but I'll see if I can find out.

**Comment by matthewhelmke** [comment]
I did, but there were merges after the PR was created, so I did another pull and that seemed to pull in the commits. I must have done something wrong.

**Comment by matthewhelmke** [comment]
Closing this as it has been replaced by #203

**Pull request closed without merging**